client/src/styles/Header.module.css

```diff
@@ -1,3 +0,0 @@
-.header {
-    gap: max(1em, 5%);
```

Comment:
This was added for the searchBar, but because we aren't using the searchBar anymore, I deleted it